builtins/amp-img.js

```diff
@@ -262,7 +252,7 @@ export class AmpImg extends BaseElement {
 
   /** @override */
   prerenderAllowed() {
-    return this.prerenderAllowed_;
+    return !this.element.hasAttribute('noprerender');
```

Comment:
Is this method called multiple times?

Reply:
Yes. It is called on every "pass"  when it is prerender phase (i.e. doesn't affect origin, or clicked amp pages). I would estimate this can happen at maximum 10x / second / component (likely much less though. I don't remember how often passes happen practically).

How expensive is `hasAttribute`? I'd be happy to lazily cache the value instead of computing each time.

Reply:
We could just cache here.

```js
return (this.prerenderAllowed_ ??= !this.element.hasAttribute('noprerender'));
```

Reply:
Sure, i'll add the caching back